src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonRequiredAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Specifies the property is required when deserializing.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false)]
+    public sealed class JsonRequiredAttribute : JsonAttribute
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="JsonRequiredAttribute"/>.
+        /// </summary>
+        /// <remarks>
+        /// <see langword="null"/> token in JSON will not trigger a validation error.
+        /// </remarks>
+        public JsonRequiredAttribute()
```

Comment:
I think this constructor can be removed? Then all comments could be moved to the type level.

Reply:
FWIW/fyi `[JsonInclude]` is the same.

Reply:
I'll follow JsonInclude pattern here. So:
- simplify comment for ctor to follow JsonInclude (`Initializes a new instance of <see cref="JsonRequiredAttribute"/>.`)
- move all the relevant info to the class comment and extend with all other comments